tests/integration/build_invoke/dotnet/test_build_invoke_dotnet8.py

```diff
@@ -0,0 +1,79 @@
+from tests.integration.build_invoke.build_invoke_base import BuildInvokeBase
+
+"""
+sam build does not support to build dotnetcore 3.1 templates using container, 
+here we put them in a separate file and use a dedicate codebuild project with .net 3.1 runtime to build them.
+
+For each template, it will test the following sam commands:
+1. sam init
+2. sam build --use-container (if self.use_container is False, --use-container will be omitted)
+3. (if there are event jsons), for each event json, check `sam local invoke` response is a valid json
+"""
+
+
+class BuildInvoke_dotnet8_cookiecutter_aws_sam_hello_dotnet(BuildInvokeBase.DotNetCoreExtraRerunBuildInvokeBase):
+    use_container = False
+    directory = "dotnet8/hello"
```

Comment:
We might want to disable linting (`should_test_lint = False`) until cfn-lint gets updated.

Reply:
I added it to all the tests, is that what you meant? I don't have enough context about cfn-lint to know more

Reply:
https://github.com/aws-cloudformation/cfn-lint is a tool for linting the CFN template for errors. They do release runtime updates after the runtime is launched, for that reason we should skip those tests to validate the template (the template is still validated but not through cfn-lint). These `should_test_lint = False` is removed once they have their release and merged into SAM CLI.